### ISSUE TITLE
controller offset

### ISSUE THREAD

**Issue by ELRandomMan** [matzman666/OpenVR-InputEmulator#208]
the controller enable offset checkbox will not check, am I doing something wrong?
Please help

**Comment by ghost** [comment]
Yes, I have it working. Had to uninstall then delete the folder and redownload from github. I then had to download a fresh dll. This is working on SteamVR 1.16.8 I use it to set the offsets for my tracker based guns.
[driver_00vrinputemulator (2).zip](https://github.com/matzman666/OpenVR-InputEmulator/files/6104488/driver_00vrinputemulator.2.zip)

**Comment by ilimo** [comment]
https://ovrmc.dschadu.de/

The project is now continued there
